compose/cli/main.py

```diff
@@ -268,6 +268,7 @@ def pull(self, project, options):
         Usage: pull [options] [SERVICE...]
 
         Options:
+            --skip-missing          Skip unpullable images.
```

Comment:
Sounds good to me. I also update parameter names to be consistent. I there somethig else, some other doc that needs to be updated as well?